docs/source/user_guide/timeseries.ipynb

```diff
@@ -328,6 +328,45 @@
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is desirable to enhance the decomposer component with a guess at the period of the seasonal aspect of the signal before decomposing it.  To do that, we can use the `determine_periodicity(X, y)` function of the `Decomposer` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "period = pdc.determine_periodicity(X_train_time, y_train_time)\n",
+    "print(period)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The children of the `Decomposer` class, if not set in the constructor, sets its `seasonal_period` parameter to -1.  If the `Decomposer` object is fit with a `seasonal_period` of -1, it will take on a default value of 7, which is good for daily data signals that have a known seasonal component period that is weekly.  \n",
```

Comment:
With respect to leaving the `seasonal_period` to the children, I initially did that because I was not entirely sure whether the other children of `Decomposer` would need this functionality or not.  I filed this [issue](https://alteryx.atlassian.net/browse/TML-5131) to come back to it after we do one or two more `Decomposer` children.

Reply:
With respect to the "default" value, I understand the confusion.  I changed the wording in the notebook as the default value is not really even 7.  "-1" means "let `statsmodels`' `freq_to_period` figure it out base on the inferred frequency of the datetime index."  Setting any other integer makes that the seasonal period.